commit 5073f41e743fe69e23806d409b23ee7293642bb7
Author: Dan McMahill
Date:   Fri Jun 5 23:49:26 2009 -0400

    Use the appropriate header files instead of providing extern foo() prototypes.
    
    Use the correct headers for a handful of function prototypes.  This should
    fix some build issues reported on geda-uers.

diff --git a/src/edif.y b/src/edif.y
index f1cb346..1bad8ff 100644
--- a/src/edif.y
+++ b/src/edif.y
@@ -17,6 +17,13 @@
  *									*
  ************************************************************************/
 #include <stdio.h>
+
+/* for malloc, free, atoi */
+#include <stdlib.h>
+
+/* for strcpy */
+#include <string.h>
+
 #include <ctype.h>
 
 #include "global.h"
@@ -34,14 +41,6 @@ LibraryEntryTypePtr GetLibraryEntryMemory (LibraryMenuTypePtr);
 #define	ABS(v)			((v) < 0 ? -(v) : (v))
 #define	Getc(s)			getc(s)
 #define	Ungetc(c)		ungetc(c,Input)
-/*
- *	External functions.
- */
-extern void *malloc();
-extern char *strcpy();
-extern int atoi();
-extern void free();
-
 
  typedef struct _str_pair
  {
--- pcb-20081128.orig/src/edif.c	2008-11-24 22:45:57.000000000 -0500
+++ pcb-20081128/src/edif.c	2009-09-09 22:50:47.000000000 -0400
@@ -686,6 +686,13 @@
  *									*
  ************************************************************************/
 #include <stdio.h>
+
+/* for malloc, free, atoi */
+#include <stdlib.h>
+
+/* for strcpy */
+#include <string.h>
+
 #include <ctype.h>
 
 #include "global.h"
@@ -703,14 +710,6 @@
 #define	ABS(v)			((v) < 0 ? -(v) : (v))
 #define	Getc(s)			getc(s)
 #define	Ungetc(c)		ungetc(c,Input)
-/*
- *	External functions.
- */
-extern void *malloc();
-extern char *strcpy();
-extern int atoi();
-extern void free();
-
 
  typedef struct _str_pair
  {
